contracts/EtherToken.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity ^0.5.2;
+// SPDX-License-Identifier: LGPL-3.0-or-newer
```

Comment:
The `package.json` specifies `LGPL-3.0` as license, which is `LGPL-3.0-only` and not `LGPL-3.0-or-newer`: https://spdx.org/licenses/